src/components/stepper.js

```diff
@@ -188,6 +193,14 @@ const StepperWithStatus = (props) => {
                 </span>
             </Grid>
             <Grid item xs={12} md={2}></Grid>
+            <ExportDialog
+                open={openExportDialog}
+                onClose={handleCloseExport}
+                onClick={handleClickExport}
+                process={props.merge === undefined ? '' : props.merge.process}
+                date={props.merge === undefined ? '' : props.merge.date}
+                title={useIntl().formatMessage({ id: 'exportNetwork' })}
```

Comment:
move hook call at the begining of the render function. This makes it easier to enforce the rules of hooks